package/jailer.sh

```diff
@@ -57,21 +57,6 @@ cp -r -l /opt/drivers/management-state/bin /opt/jail/$NAME/var/lib/rancher/manag
 # Hard link rancher-machine into the jail
 cp -l /usr/bin/rancher-machine /opt/jail/$NAME/usr/bin
 
-# Hard link helm_2 into the jail
-cp -l /usr/bin/rancher-helm /opt/jail/$NAME/usr/bin
-
-# Hard link helm_3 into the jail
-cp -l /usr/bin/helm_v3 /opt/jail/$NAME/usr/bin
-
-# Hard link tiller into the jail
-cp -l /usr/bin/rancher-tiller /opt/jail/$NAME/usr/bin
-
-# Hard link kustomize into the jail
-cp -l /usr/bin/kustomize /opt/jail/$NAME/usr/bin
-
-# Hard link custom kustomize script into jail
-cp -l /usr/bin/kustomize.sh /opt/jail/$NAME
```

Comment:
This is potentially used by Node drivers. I would probably leave `kustomize{,.sh}` and `helm_v3`.

In doubt, it may be worth reaching out to hostbusters to confirm.

Reply:
We shoud keep the `kustomize`, `kustomize.sh`, and `helm_v3` used for managing Catalog App. Rancher runs the operation in a jail.